src/NzbDrone.Core/Extras/Subtitles/SubtitleService.cs

```diff
@@ -96,7 +96,12 @@ public override ExtraFile Import(Movie movie, MovieFile movieFile, string path,
             if (SubtitleFileExtensions.Extensions.Contains(Path.GetExtension(path)))
             {
                 var language = LanguageParser.ParseSubtitleLanguage(path);
-                var suffix = GetSuffix(language, 1, false);
+                var subtitleFiles = _subtitleFileService.GetFilesByMovie(movie.Id);
+                var existingSrtSubs = subtitleFiles.Where(m => m.MovieFileId == movieFile.Id)
+                    .Where(m => m.Language == language)
+                    .Where(m => m.Extension == extension);
+
+                var suffix = GetSuffix(language, existingSrtSubs.Count() + 1, extension.EqualsIgnoreCase(".srt"));
```

Comment:
Suffix subtitle files with an index to support importing multiple version of subtitle files (2_English.srt, 3_English.srt)